### PR TITLE
chore: fix focus style

### PR DESCRIPTION
**Pull request by kendallstrautman**
## 🔗 Relevant links

- [Preview link](https://dev-portal-git-ksfixup-focus-ring-hashicorp.vercel.app/waypoint) 🔎
- [Asana task](https://app.asana.com/0/1203652232454021/1204304009719187) 🎟️

## 🗒️ What

This fixes up the focus ring styles that were being overridden by the dark override styles for the `box-shadow`. 

Screenshot 2023-03-30 at 5 15 57 PM

## 🧪 Testing

- [Visit the preview ](https://dev-portal-git-ksfixup-focus-ring-hashicorp.vercel.app/waypoint), enable dark theme and tab through the nav. The command bar activator input should have focus styles that match light mode.

### CODE REVIEW

File: src/components/command-bar/components/activator/command-bar-activator.module.css [hunk 1]
```diff
@@ -47,6 +47,12 @@
 
 		/* TODO update the light version with a token instead of manual box shadow */
 		box-shadow: unset;
```
Comment (kendallstrautman) [inline src/components/command-bar/components/activator/command-bar-activator.module.css:49]: The focus styles were being overridden by this line.

File: src/components/command-bar/components/activator/command-bar-activator.module.css [hunk 2]
```diff
@@ -47,6 +47,12 @@
...
+
+		/* Ensures the g-focus-ring-from-box-shadow-dark styles aren't overridden */
+		&:focus-visible {
```
Comment (kendallstrautman) [inline src/components/command-bar/components/activator/command-bar-activator.module.css:52]: I can't use `compose` in here, so I just copied the styles from `g-focus-ring-from-box-shadow-dark`

**Comment by kendallstrautman** [comment]
@ashleemboyer Let me know your thoughts on this approach. We updated both light and dark mode to use the same border so that the box-shadow overrides aren't messed with and the focus styles apply consistently between themes 🙇🏻‍♀️

**Review by ashleemboyer — APPROVED** [review]
The update is awesome! 🙌🏻

- Got rid of a pesky `data-theme` override ✅ 
- Continuing to leverage `g-focus-ring-from-box-shadow-dark` global helper ✅ 
- AND got rid of a hard-coded `box-shadow` color? 😍 ✅

**Review by braydenlove — APPROVED** [review]
WOOOOO!

**Pull request merged**